deploy/README.md

```diff
@@ -6,8 +6,12 @@ server to serve connection information to DAQ applications.
 ## Installation
 
  Build the docker image
+```bash
+docker buildx build --tag ghcr.io/dune-daq/connectivityserver:latest .
```

Comment:
Maybe add a note on the `VERSION` arg?

Reply:
Yes, it's the next block, so I think it's done. But I'll add a comment saying that it will build from develop